src/os/ui/text/tuieditor.js

```diff
@@ -287,6 +287,12 @@ os.ui.text.TuiEditorCtrl.prototype.getOptions = function() {
     options['toolbarItems'] = toolbarItems;
   }
 
+  const plugins = this.getPlugins();
+
+  if (plugins instanceof Array && plugins.length > 0) {
```

Comment:
This might be better as `Array.isArray(plugins)`

Reply:
The first check has been updated to use `Array.isArray()`

Reply:
Actually, it looks to work just fine with empty arrays. I changed it to just be the one check.